### PR TITLE
Fix: songs looping at wrong times

### LINKED ISSUE #68: The town tune throws off the loop times.
Once the town tune plays, it throws off the loop times for the songs, causing jarring loops.

### PR DESCRIPTION
**Pull request by PikaDude**
Closes #68. The issue was is that the tab audio handler would pause/unpause the music during the town tune, causing it to spawn multiple loop time timeouts and other various issues. Also updated changelog.

### CODE REVIEW

File: options.html [hunk 1]
```diff
@@ -256,11 +256,15 @@ <h1>Changelog</h1>
 				</header>
 
 				<div class="content">
-					<h3>Version 4.0.1<span class="changelog-date"> - 12th of February 2020</span></h3>
+					<h3>Version 4.0.1<span class="changelog-date"> - ??th of February 2020</span></h3>
```
Comment (jordigarcl) [inline options.html:259]: Careful, this got reverted to `??`

Comment (PikaDude) [reply options.html:259]: Yeah, I've decided that we should probably set this properly last, right when we're about to merge to master.

**Review by jordigarcl — COMMENTED** [review]
I haven't had a chance to test it but logic makes sense to me. Is this something to be tested against all browsers?

**Review by jordigarcl — APPROVED** [review]
Approving this since it doesn't look to have any major bugs and don't want to block the release

**Review by paladique — APPROVED** [review]

**Comment by PikaDude** [comment]
Yeah maybe test this pr on all browsers just to be sure

**Pull request merged**